build/cqs.tsv

```diff
@@ -76,6 +80,8 @@ io.airlift:aircompressor	0.10	compile
 io.dropwizard.metrics:metrics-core	3.2.6	compile
 io.dropwizard.metrics:metrics-ganglia	3.2.6	compile
 io.dropwizard.metrics:metrics-graphite	3.2.6	compile
+io.dropwizard.metrics:metrics-jvm	4.0.2	compile
```

Comment:
this is going to need to match the rest of the metrics versions too. @jahhulbert-ccri just asked us to bump the metrics version - looks like 4.1.0 is the latest. Could you try setting the version to that, and see if things still build ok? We'll also need to add an entry for metrics-jvm here: https://github.com/locationtech/geomesa/blob/master/pom.xml#L1584

Reply:
When I changed the metrics version I get the following error during the build: Could not resolve dependencies for project org.locationtech.geomesa:geomesa-metrics_2.11:jar:2.4.0-SNAPSHOT: Failure to find io.dropwizard.metrics:metrics-ganglia:jar:4.1.0 in http://art.ccri.com/artifactory/libs-release was cached in the local repository, resolution will not be reattempted until the update interval of libs-release has elapsed or updates are forced

Reply:
ok, thanks - to not hold up this PR, can you just set the metrics-jvm version to 3.2.6, and we can update everything to 4.x later.